Classes/Driver/Version2/IndexerDriver.php

```diff
@@ -124,7 +124,7 @@ public function fulltext(NodeInterface $node, array $fulltextIndexOfNode, $targe
                     'inline' => '
                         ctx._source.__fulltext = new HashMap();
                         
-                        if (!ctx._source.containsKey("__fulltextParts")) {
+                        if (!(ctx._source.containsKey("__fulltextParts") && ctx._source.__fulltextParts instanceof HashMap)) {
```

Comment:
can we use instanceof "Map" here? Would be a little more stable and in line with Java APIs.